src/main/java/uk/gov/hmcts/reform/civil/handler/callback/user/RespondToClaimSpecCallbackHandler.java

```diff
@@ -96,6 +97,20 @@ private CallbackResponse handleDefendAllClaim(CallbackParams callbackParams) {
         }
         if ("DEFENDANT_RESPONSE_SPEC".equals(callbackParams.getRequest().getEventId())) {
 
+            if ("HAS_PAID_THE_AMOUNT_CLAIMED".equals(caseData.getDefenceRouteRequired())
```

Comment:
Please can you write some tests to cover this logic. You may follow existing test styles around this part of the code

Reply:
can we add this bit for in a future capacity? as this was from the last sprint